sympy/ntheory/ecm.py

```diff
@@ -3,6 +3,8 @@
 from sympy.core.compatibility import as_int
 import random
 
+prng = random.Random()
```

Comment:
the name can be better, like: `randgen` or `rgen`

Reply:
Done.